lib/gooddata/lcm/helpers/check_helper.rb

```diff
@@ -51,8 +51,6 @@ def collect_synced_status(params)
           params.include?(COLLECT_SYNCED_STATUS) && to_bool(COLLECT_SYNCED_STATUS, params[COLLECT_SYNCED_STATUS])
         end
 
-        private
```

Comment:
irrelevant change?

Reply:
No, It relates to the change. Remove it to public method below it.